framework/helpers/BaseConsole.php

```diff
@@ -358,32 +358,32 @@ public static function ansiStrlen($string)
     public static function ansiToHtml($string, $styleMap = [])
     {
         $styleMap = [
-            // http://www.w3.org/TR/CSS2/syndata.html#value-def-color
-            self::FG_BLACK => ['color' => 'black'],
```

Comment:
Array lists should not be indented, since all lines will change, if just one longer entry is added.

Reply:
Thank, forget about this.